tensorboard/components/tf_data_selector/tf-data-selector.ts

```diff
@@ -40,55 +53,47 @@ Polymer({
       value: () => [],
     },
 
-    runSelectionState: {
-      type: Object,
-      observer: '_storeRunSelectionState',
-      value: () => tf_storage.getObject('runSelectionState') || {},
-    },
-
-    runRegexInput: {
+    _addLabel: {
       type: String,
-      value: tf_storage.getStringInitializer('regexInput', {defaultValue: ''}),
-      observer: '_runRegexInput',
+      computed: '_getAddLabel(_selectedExperiments)',
     },
   },
 
   get expsStore() {
     return tf_backend.experimentsStore;
   },
 
-  get runsStore() {
-    return tf_backend.runsStore;
-  },
-
   attached() {
     this._updateExpKey = this.expsStore.addListener(() => this._updateExps());
-    this._updateRunKey = this.runsStore.addListener(() => this._updateRuns());
-
     this._updateExps();
-    this._updateRuns();
   },
 
   detached() {
     this.expsStore.removeListenerByKey(this._updateExpKey);
-    this.runsStore.removeListenerByKey(this._updateRunKey);
   },
 
   _updateExps() {
     const expNames = this.expsStore.getExperiments().map(({name}) => name);
-    this.set('experiments', expNames);
+    this.set('_experiments', expNames);
   },
 
-  _updateRuns() {
-    const runNames = this.runsStore.getRuns();
-    this.set('runs', runNames);
+  _toggleExperimentAdder() {
+    this._experimentsAdding = !this._experimentsAdding;
```

Comment:
Optional, but I think `showExperimentAdder` would be a little clearer as a property name